other/cmr-exchange/graph/project.clj

```diff
@@ -38,6 +38,7 @@
                  [metosin/ring-http-response "0.9.1"]
                  [org.clojure/clojure "1.9.0"]
                  [org.clojure/data.csv "0.1.4"]
+                 ;; TODO Should these keep up with other dependencies?
```

Comment:
same thing these were not kept up with. If that's fine I will just remove the comment.